distributed/utils_test.py

```diff
@@ -211,6 +212,57 @@ def slowidentity(*args, **kwargs):
         return args
 
 
+def varying(items):
```

Comment:
I was surprised I had to use such an elaborate hack for this use case. Do you know of a simpler way?

Reply:
No, not really.  I haven't thought about this much.  Though elaborate, it does appear to get the job done :)